frontend/src/pages/modelServing/screens/global/EmptyModelServing.tsx

```diff
@@ -19,14 +16,9 @@ const EmptyModelServing: React.FC = () => {
     project,
   } = React.useContext(ModelServingContext);
   const servingPlatformStatuses = useServingPlatformStatuses();
-  const isKServeNIMEnabled = project ? isProjectNIMSupported(project) : false;
 
-  if (
-    (getProjectModelServingPlatform(project, servingPlatformStatuses).platform !==
-      ServingRuntimePlatform.SINGLE ||
-      isKServeNIMEnabled) &&
-    servingRuntimes.length === 0
-  ) {
+  if (servingPlatformStatuses.modelMesh.enabled && servingRuntimes.length === 0) {
```

Comment:
Is removing `|| isKServeNIMEnabled` from this logic desired? Seems like yes but I'm not sure.

Reply:
Yeah, I think it was a gap -- I could put it back for sure -- but NIM is just KServe so deploying the modal should be the same process -- I'll go double check this flow carefully to make sure. Thanks for calling it out.

Reply:
I think you're right, this was unnecessarily directing the user to the project which is necessary for model servers, but not for KServe/NIM